CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## [TBD]
 ### Added
 * Regex support in `federated-meta-stores.mapped-databases`. [#59](https://github.com/HotelsDotCom/waggle-dance/issues/59).
+### Changed
+* Replace SSH support with _hcommon-ssh_ library. [#51](https://github.com/HotelsDotCom/waggle-dance/issues/51).
```

Comment:
Make "hcommon-ssh" link to the GH page for the library so it's clear where this has gone to. We should do a major version upgrade for this since we moved so many classes.

Reply:
done